doc/src/site/sphinx/deployment/load_mojo.rst

```diff
@@ -261,6 +261,8 @@ We can configure the output and format of predictions via the H2OMOJOSettings. T
 - ``withContributions`` - Enables or disables computing Shapley values. Shapley values are generated as a sub-column for the
   detailed prediction column. Therefore, to compute Shapley values, both this option and ``withDetailedPredictionCol`` needs to be
   enabled. By default, it is disabled.
+- ``leafNodeAssignmentsEnabled`` - When enabled, user can obtain the leaf node assignments after the model traininig
```

Comment:
Why this flag is needed and where it's used internally?

Reply:
Good question -> It is used within `EasyPredictModelWrapper` in predict methods, for example in `predictBinomial`:

```
    if (enableLeafAssignment) { // only get leaf node assignment if enabled
      SharedTreeMojoModel.LeafNodeAssignments assignments = leafNodeAssignmentExtended(data);
      p.leafNodeAssignments = assignments._paths;
      p.leafNodeAssignmentIds = assignments._nodeIds;
    }
```

But I actually messed these up in the code -> the method leafNodeAssignments should not take df as input and we should just return the calculaded assignments when this option was enabled.

Thanks, will fix